app/views/targets/_info.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.id target.id
+json.topic target.topic, partial: 'topics/info', as: :topic
```

Comment:
You have an N+1 situation in here, check this [link](https://medium.com/@gsanchezd/el-problema-de-las-n-1-queries-de-ruby-on-rails-4d30ad6119a)